ldns/util.h.in

```diff
@@ -274,19 +274,19 @@ time_t mktime_from_utc(const struct tm *tm);
 
 /**
  * The function interprets time as the number of seconds since epoch
- * with respect to now using serial arithmitics (rfc1982).
+ * with respect to now using serial arithmetics (rfc1982).
```

Comment:
maybe `arithmetic`?

Reply:
yes, but it is a public facing symbol, so we need to maintain the old name too.